staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -350,7 +95,12 @@ func (s *SpdyRoundTripper) NewConnection(resp *http.Response) (httpstream.Connec
 		return nil, fmt.Errorf("unable to upgrade connection: %s", responseError)
 	}
 
-	return NewClientConnectionWithPings(s.conn, s.pingPeriod)
+	rwc, ok := resp.Body.(io.ReadWriteCloser)
```

Comment:
can we add a comment explaining that the Upgrade request was successful at this point and the response body should be bidirectional?